worker/jobs/bulkUpsertSamplesJob.js

```diff
@@ -60,6 +60,14 @@ module.exports = (job, done) => {
         }
       }
 
+      // force garbage collection after samples are published
+      if (global.gc) {
+        global.gc();
+      } else {
+        logger.info('Garbage Collection unavailable. Pass --expose-gc when ' +
```

Comment:
I don't think we need this logged on every job, esp. if this is just experimental. If we like the way this works, I'd feel better putting it behind a toggle, e.g. `if (featureToggles.isFeatureEnabled(...) && global.gc) { ... }`

Reply:
I was thinking we would run a clock job just for this if this works. Not sure what is the right way to do it though.